CMakeLists.txt

```diff
@@ -1,31 +1,68 @@
-# NOTE:  If you installed SDL2 at an unusual place and FindSDL2 fails,
-#        you can define SDL2_INCLUDE_DIR on the cmdline.  For example:
-#        cmake -DSDL2_INCLUDE_DIR=/opt/SDL2/include/SDL2 [other stuff]
+# NOTE:  If you installed SDL3 at an unusual place and SDL3Config fails,
+#        you can define SDL3_INCLUDE_DIRS on the cmdline.  For example:
+#        cmake -DSDL3_INCLUDE_DIRS="/opt/SDL3/include;/opt/SDL3/include/SDL3"
```

Comment:
Minor nit: The `/SDL3` subdir isn't needed anymore, so the line can simply read:
`#        cmake -DSDL3_INCLUDE_DIRS=/opt/SDL3/include`